test/Data/Swagger/ParamSchemaSpec.hs

```diff
@@ -30,6 +34,7 @@ spec = do
     context "Unary records" $ do
       context "Email (unary record)"  $ checkToParamSchema (Proxy :: Proxy Email)  emailSchemaJSON
       context "UserId (non-record newtype)" $ checkToParamSchema (Proxy :: Proxy UserId) userIdSchemaJSON
+    context "TimeOfDay" $ checkToParamSchema (Proxy :: Proxy TimeOfDay) (Object (HM.fromList [("format",String "hh:MM:ss"),("type",String "string")]))
```

Comment:
Could you put schema's JSON value in `SpecCommon`, just like `colorSchemaJSON` and others?

Reply:
done